docs/Graph Data Format.md

```diff
@@ -0,0 +1,85 @@
+## VisualDL Graph data format design
+
+## Backgroud
+Neural network have a concept of `Model`. VisualDL has a very import feature that it can visualize the structure of deep learning `Model`. As an visualize tool that want to support many different deeplearning framework, we should use a generic Model data format that is supported by most of frameworks.
+
+Facebook opensourced a [ONNX](http://onnx.ai/)(Open Neural Network Exchange). ONNX is a open format to represent deep learning models. It now officially supported by world's most popular framework like Tensorflow, Caffe2, MxNet, PaddlePaddle Fluid, so we decided to choose ONNX as the standard Model format and support the visualization of ONNX model.
```

Comment:
a -> an?

MxNet -> MXNet

> It now officially supported by world's most popular framework

Currently, it is officially supported by most of the popular frameworks in the world.